utility/src/main/java/org/oppia/android/util/accessibility/BUILD.bazel

```diff
@@ -0,0 +1,66 @@
+"""
+General purposes utilities for Android accessibility checking.
+"""
+
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+
+kt_android_library(
+    name = "accessibility",
+    srcs = [
+        ":AccessibilityChecker.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
+)
+
+kt_android_library(
+    name = "impl",
+    srcs = [
+        ":AccessibilityCheckerImpl.kt",
+    ],
+    visibility = ["//visibility:private"],
```

Comment:
Do we need to specify this as it is private by default?

Reply:
Nope--good point. I'll remove this when I address the other code-based changes.

Reply:
Done.